src/test/test_bitcoin.cpp

```diff
@@ -25,7 +25,7 @@
 
 CClientUIInterface uiInterface; // Declared but not defined in ui_interface.h
 CWallet* pwalletMain;
-libzerocash::ZerocashParams *pzerocashParams;
+ZCJoinSplit *pzcashParams;
```

Comment:
There is no duplicated global variable; `init.cpp` is not linked into the test framework, but `init.h` declares an external `zcashParams`, which the test framework provides. I think the test framework could eventually initialize it using a generated circuit -- for now it does nothing. However, it will cause linker errors if we don't declare it.